### ISSUE TITLE
Comparison with the 2018 study by the same authors

### ISSUE THREAD

**Issue by tpellard** [lexibank/robbeetstriangulation#9]
Robbeets et al (2021) barely mention the previous study by Robbeets & Bouckaert (2018) on Transeurasian phylogeny:
https://academic.oup.com/jole/article/3/2/145/5067185

They do not compare their new results with the previous ones and do not mention that they use a different cognate coding method. In 2018, they considered words from the same root but with a different meaning to be cognates, but in 2021 they only include words with the same meaning and a shared root.

The resulting tree of 2018 is different, with a 100% posterior probability for a Turkic-Mongolic node, (Fig. 8) while in 2021 there is a Monlogic-Tungusic node instead (lines 112-113, Fig. 1b., I couldn't find the "extended data"). I compiled below a summary of the differences between the two studies (I couldn't find all the info, please add it if you can).

|                  | 2018                    | 2021                    |
|------------------|-------------------------|-------------------------|
| nforms           |                         | 3193                    |
| nconcepts        | 150?                    | 254                     |
| nlects           | 57                      | 98                      |
| tree             | (((Tk, M), Tg), (K, J)) | ((Tk, (M, Tg)), (K, J)) |
| pTEA             | ?6650 BP (4330-9450)    | 9181 BP (5595-12793)    |
| pTkMTg           |                         | 6811 BP (4404-10166)    |
| pMTg             |                         | 4491 BP (2599-6373)     |
| pJapano-Koreanic |                         | 5458 BP (3335-8024)     |

The following quotes from the 2018 article are I think interesting:

> Lexicostatistic dating methods such as Blažek and Schwarz (2009) reach 4750 BC for the timedepth of Transeurasian, a date, which converges with our own Bayesian estimation of the root age of Transeurasian as 4700 BC. As the latter timedepth does not substantially differ from that of our prior, we cannot draw firm conclusions on the dating or proto-Transeurasian on the basis of Bayesian inference alone

So basically their result was just their prior?

> Unfortunately, as far as timing is concerned, the root age of the inferred phylogeny has large uncertainty when using a Yule prior (95% HPD Interval [2.55kya, 7.54kya] a priori [4.33kya, 9.45kya] a posteriori), so we cannot draw firm conclusions on the timing aspects of our results.

But isn't the interval for pTEA in the 2021 study (5595-12793) even larger?
